src/v/utils/hdr_hist.h

```diff
@@ -45,6 +45,8 @@ inline hdr_histogram_ptr make_unique_hdr_histogram(
 // Potentially VERY expensive object. At default granularity is about 4k bytes
 class hdr_hist {
 public:
+    static constexpr int64_t us_per_hour = 3600000000;
```

Comment:
is it supposed to look like `μs`?

Reply:
yes, but we should probably stick with the ascii :)

Reply:
https://mariusbancila.ro/blog/2019/05/16/cpp-is-fun/